releases/v0.12.md

```diff
@@ -0,0 +1,74 @@
+# v0.12
+
+This document contains all release notes pertaining to the `v0.12.x` releases of Vela.
+
+## For Vela Administrators
+
+* When upgrading from `v0.11`, please take note of the [migration information](/migrations/v0.12/README.md)
+
+## Full release notes available on [github.com/go-vela](https://github.com/go-vela)
+
+* [go-vela/ui](https://github.com/go-vela/ui/releases)
+* [go-vela/server](https://github.com/go-vela/server/releases)
+* [go-vela/worker](https://github.com/go-vela/worker/releases)
+* [go-vela/cli](https://github.com/go-vela/cli/releases)
+* [go-vela/sdk-go](https://github.com/go-vela/sdk-go/releases)
+* [go-vela/types](https://github.com/go-vela/types/releases)
+
+## v0.12.0
+
+Happy February Vela Community!
+
+Below is a summary of what's new in 0.12.0
+
+### Breaking Changes
+
+* (types) fix(secret)!: remove implicit repo secrets and clean up secrets testing by @GregoryDosh in https://github.com/go-vela/types/pull/210
+  - existing implicit secrets usage will cease to work
+* (server) feat(repo)!: add build rate limiting by @jbrockopp in https://github.com/go-vela/server/pull/558
+  - introduces new behavior
+* (server) feat(repo/webhook)!: handle repository rename in database by @ecrupper in https://github.com/go-vela/server/pull/556
+  - depends on a new DB field
+* (server) feat(secrets)!: adding new metadata to secrets by @ecrupper in https://github.com/go-vela/server/pull/526
+  - depends on new DB fields
+
+### Bug Fixes
+
+* (cli) fix(validate): handle .yaml default file by @wass3r in https://github.com/go-vela/cli/pull/326
```

Comment:
Just a formatting comment, I personally like what was done [here](https://github.com/go-vela/community/blob/fde93e0c039dbb923949621fb4d6c4a43b569e2f/releases/v0.11.md#bug-fixes-1) vs the raw link.

Reply:
fair.. i pulled these directly from the GitHub generated release notes, hence the difference. I will adjust to adapt the previous format for this. incoming..

Reply:
Really liked the `@<user>` for call outs! I remember us trying to think of ways of doing that in the past.

Reply:
made the PR reference edit

Reply:
@kneal yes, i liked that as well.. i think where it gets a bit tricky is that to be inclusive we are leaving out folks that contribute in other ways. whether that's bug reports, feature requests, reviewing, discussion participants, etc. - kind of in the spirit of https://github.com/all-contributors. i think the current form of this presented here is not final form.. maybe it'll be closer to `all-contributors`?

Reply:
Yeah, that's true. 😞 

There's definitely a lot more to a feature than just being the code author.